server/test/unit/server/db/test_model.py

```diff
@@ -590,3 +590,30 @@ def test_meta_collection(self):
         Assert that the collection name is correct.
         """
         self.assertEquals(model.CeleryBeatLock._meta['collection'], 'celery_beat_lock')
+
+
+class TestDeferredDownload(unittest.TestCase):
+    """
+    Test the DeferredDownload class.
+    """
+
+    def test_model_superclass(self):
+        sample_model = model.DeferredDownload()
+        self.assertTrue(isinstance(sample_model, Document))
+
+    def test_attributes(self):
+        self.assertTrue(isinstance(model.DeferredDownload.unit_id, StringField))
+        self.assertTrue(model.DeferredDownload.unit_id.required)
+
+        self.assertTrue(isinstance(model.DeferredDownload.unit_type_id, StringField))
+        self.assertTrue(model.DeferredDownload.unit_type_id.required)
+        self.assertEqual(['unit_id'], model.DeferredDownload.unit_type_id.unique_with)
+
+        self.assertTrue(isinstance(model.DeferredDownload._ns, StringField))
+        self.assertEqual('deferred_download', model.DeferredDownload._ns.default)
+
+    def test_meta_collection(self):
+        """
+        Assert that the collection name is correct.
+        """
+        self.assertEquals(model.DeferredDownload._meta['collection'], 'deferred_download')
```

Comment:
It would be good to make less assertions on `_meta`. One way this cold be done using public methods would be to make assertions on the response output from `list_indexes`. Consider this snippet from your model definition.

```
DeferredDownload.list_indexes()
[[('unit_id', 1), ('unit_type_id', 1)], [(u'_id', 1)]]
```